srcpkgs/libchewing/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'libchewing'
+pkgname=libchewing
+version=0.5.1
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="autoconf automake libtool clang gcc"
```

Comment:
gcc is part of the base-system, remove it.

Reply:
auto* is part of base-devel, does it need to go too?